lib/manifestTools/assets/web-manifest.json

```diff
@@ -37,7 +37,12 @@
       "description": "Represents the URL that the developer would prefer the user agent load when the user launches the web application.",
       "type": "string",
       "format": "uri"
+    },
+    "theme_color": {
+      "description": "background color for splash screen and icons",

```

Comment:
Please note that theme color generally affects the chrome of the application - we are hoping to have background colors specifically for splashscreen and icons.